doc/source/whatsnew/v0.20.0.txt

```diff
@@ -384,3 +384,5 @@ Bug Fixes
 - Bug in ``pd.read_csv()`` for the C engine where ``usecols`` were being indexed incorrectly with ``parse_dates`` (:issue:`14792`)
 
 - Bug in ``Series.dt.round`` inconsistent behaviour on NAT's with different arguments (:issue:`14940`)
+
+- Bug in ``pd.read_csv()`` with ``float_precision='round_trip'`` which caused a segfault when a text entry is parsed on Python 2.7 or later
```

Comment:
add the issue number; remove Python 2.7 or later (we in fact don't support < 2.7)